lens/src/main/resources/interpreter-setting.json

```diff
@@ -0,0 +1,51 @@
+[
+  {
+    "group": "lens",
+	"name": "lens",
+	"className": "org.apache.zeppelin.lens.LensInterpreter",
+	"properties": {
+      "zeppelin.lens.run.concurrent": {
+        "envName": null,
+		"propertyName": "zeppelin.lens.run.concurrent",
+		"defaultValue": "true",
+		"description": "Run concurrent Lens Sessions"
+	  },
+	  "zeppelin.lens.maxThreads": {
+		"envName": null,
+		"propertyName": "zeppelin.lens.maxThreads",
+		"defaultValue": "10",
+		"description": "If concurrency is true then how many threads?"
+	  },
+	  "zeppelin.lens.maxResults": {
+        "envName": null,
+   	    "propertyName": "zeppelin.lens.maxResults",
+	    "defaultValue": "1000",
+	    "description": "max number of rows to display"
+	  },
+	  "lens.server.base.url": {
+		"envName": null,
+		"propertyName": "lens.server.base.url",
+		"defaultValue": "http://<hostname>:<port>/lensapi",
+		"description": "The URL for Lens Server"
+	  },
+	  "lens.client.dbname": {
+		"envName": null,
```

Comment:
This line is 2 tabs, and the above one is 2 spaces and 1 tab. Can you change every thing in multiple of 2 spaces. 

Please make sure that ALL your Java code follows the conventions from [project style guide](https://zeppelin.apache.org/contribution/contributions#code-convention).